components/nexus-repository/src/main/java/org/sonatype/nexus/repository/view/ContentTypes.java

```diff
@@ -19,15 +19,28 @@
  */
 public class ContentTypes
 {
+  /**
+   * Plain text.
+   */
   public static final String TEXT_PLAIN = "text/plain";
 
+  /**
+   * Generic HTML content type.
+   */
   public static final String TEXT_HTML = "text/html";
 
-  public static final String TEXT_XML = "text/xml";
```

Comment:
I think it should be removed as Tika defines "text/xml" as _alias of_ "application/xml". Also, according to some SO answers, those two are basically synonyms of each other, with maybe one rightful difference: "text/xml" is XML that is readable by humans, while "application/xml" is XML that has no such intent. 

So, I just wanted to avoid problem of tinkering when someone needs content type, to decide _which one to use_. If we have one XML content type, there is no such doubt.